src/main/java/com/google/growpod/servlets/UserServlet.java

```diff
@@ -127,4 +131,20 @@ public UserDao getDao() {
   public void setDao(UserDao dao) {
     this.dao = dao;
   }
+
+  /**
+   * Currently this Post function will get the user data in the form of a Json object,turn the Json
```

Comment:
Nit: too many of the gory details in the comment. It is better to stay high level unless something confusing is happening. E.g.: "Processes a request to add a new user"

Reply:
Done!